# HG changeset patch
# User Rik <rik@octave.org>
# Date 1526533390 25200
#      Wed May 16 22:03:10 2018 -0700
# Node ID dd77a0628ccea1feaed2b3d0dae6b704567bb6e1
# Parent  9b67c22941294df901f412cc67fac3118dd0781a
# Parent  2b0680bdb1ed418b78cee63985736cae5fefe565
maint: merge stable to default.

diff --git a/libgui/graphics/__init_qt__.cc b/libgui/graphics/__init_qt__.cc
--- a/libgui/graphics/__init_qt__.cc
+++ b/libgui/graphics/__init_qt__.cc
@@ -61,48 +61,16 @@ namespace QtHandles
 
             octave::gtk_manager& gtk_mgr = interp.get_gtk_manager ();
 
             graphics_toolkit tk (new Backend ());
             gtk_mgr.load_toolkit (tk);
 
             octave::interpreter::add_atexit_function ("__shutdown_qt__");
 
-            // Change some default settings to use Qt default colors
-            QPalette p;
-            graphics_object root = gh_manager::get_object (0);
-
-            /*
-            root.set ("defaultfigurecolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Window))));
-            */
-            root.set ("defaultuicontrolbackgroundcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Window))));
-            root.set ("defaultuicontrolforegroundcolor",
-                      octave_value (Utils::toRgb
-                                    (p.color (QPalette::WindowText))));
-            root.set ("defaultuipanelbackgroundcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Window))));
-            root.set ("defaultuipanelforegroundcolor",
-                      octave_value (Utils::toRgb
-                                    (p.color (QPalette::WindowText))));
-            root.set ("defaultuipanelhighlightcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Light))));
-            root.set ("defaultuipanelshadowcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Dark))));
-            root.set ("defaultuibuttongroupbackgroundcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Window))));
-            root.set ("defaultuibuttongroupforegroundcolor",
-                      octave_value (Utils::toRgb
-                                    (p.color (QPalette::WindowText))));
-            root.set ("defaultuibuttongrouphighlightcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Light))));
-            root.set ("defaultuibuttongroupshadowcolor",
-                      octave_value (Utils::toRgb (p.color (QPalette::Dark))));
-
             qtHandlesInitialized = true;
 
             return true;
           }
         else
           error ("__init_qt__: QApplication object must exist.");
       }
 
